library/src/main/java/com/leinardi/android/speeddial/UiUtils.java

```diff
@@ -83,21 +83,25 @@ static int pxToDp(float px) {
      * @param view view to animate.
      */
     public static void fadeOutAnim(final View view) {
-        ViewCompat.animate(view).cancel();
+        ViewCompat.animate(view).cancel
+                ();
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.whitespace.MethodParamPadCheck
**Severity**: ERROR
**File**: library/src/main/java/com/leinardi/android/speeddial/UiUtils.java L87

'(' should be on the previous line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<1783443216>*